recipe-box-frontend/src/main/frontend/src/components/HomePage.css

```diff
@@ -0,0 +1,41 @@
+
+.home-page {
+  text-align: center;
```

Comment:
I think this is not necessary, try and remove it together with the other `text-align: left;` from `.home-page h2` and see which ne you really need and if you need them both